Text_to_Translated_Text_Step_2.ipynb

```diff
@@ -0,0 +1,360 @@
+        "print(\"Google Translate : %s seconds\" % (time.time() - start_time))"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "nA9RG_GR3ZfD",
+        "outputId": "5fe3e2a7-5a54-46a4-ef26-1c04d87c39e4"
+      },
+      "execution_count": 127,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "¡Hola! Mi nombre es Fil. He sido tutor de inglés en línea durante casi tres años. He estado ayudando a estudiantes desde nivel principiante hasta avanzado a mejorar sus habilidades en inglés. He intentado enseñar a niños y adultos. Utilicé RCP. ¡Mirar! ¡Recompensas! ¡Y muchos más! Me encantaría ayudarte a mejorar y progresar en el aprendizaje. Desde fonética, gramática, pronunciación y hasta inglés conversacional. Ya obtuve mi certificado de Borla y Tia Felt, así que estás en buenas manos. Aprende conmigo y hagamos del inglés un idioma fácil y divertido. ¡Nos vemos!\n",
+            "Google Translate : 0.3525381088256836 seconds\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##Translating Text to Spanish using Azure AI Translator"
+      ],
+      "metadata": {
+        "id": "FPn55iJzEQ7k"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Install the 'requests' and 'uuid' libraries\n",
+        "!pip install requests uuid"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "w0dZrIteLS1K",
+        "outputId": "049de07f-fe48-47a0-af68-b291bc64f447"
+      },
+      "execution_count": 128,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (2.31.0)\n",
+            "Requirement already satisfied: uuid in /usr/local/lib/python3.10/dist-packages (1.30)\n",
+            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests) (3.3.0)\n",
+            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests) (2.10)\n",
+            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests) (2.0.7)\n",
+            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests) (2023.7.22)\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Import required modules\n",
+        "import requests\n",
+        "\n",
+        "start_time = time.time()\n",
+        "\n",
+        "# Define your Azure Cognitive Services Translator API credentials and configuration\n",
+        "key = \"ebe0efc466cd4c119e987d8fb0a7c748\"\n",
```

Comment:
Will this be a security concern, since the GitHub repo is a public one?

Reply:
I realized that it could be a security concern, so I have decided to remove it from the code. I will personally share the API key with my group members so they can use this code and execute it on their own. Please check the updated changes.